packages/doxygen.rb

```diff
@@ -3,18 +3,18 @@
 class Doxygen < CMake
   description 'Doxygen is the de facto standard tool for generating documentation from annotated C++ sources, but it also supports other popular programming languages such as C, Objective-C, C#, PHP, Java, Python, IDL (Corba, Microsoft, and UNO/OpenOffice flavors), Fortran, VHDL, Tcl, and to some extent D.'
   homepage 'https://www.doxygen.nl/'
-  version '1.11.0'
+  version '1.13.0'
   license 'GPL-2'
   compatibility 'all'
   source_url 'https://github.com/doxygen/doxygen.git'
   git_hashtag 'Release_1_11_0'
```

Comment:
This hasn't been updated, unless i'm missing something, so we just rebuilt 1.11.0

Reply:
https://reproducible-builds.org/ would help with these cases so there won't be new hashes and it'll be easier to detect these

Reply:
We have new binaries uploaded.